Jenkinsfile

```diff
@@ -23,21 +28,37 @@ pipeline {
 
       }
     }
-    stage('Build Kernel and package') {
+
+    stage('Cleanup') {
       steps {
-        sh 'sudo ./make_var_mx6ul_dart_debian.sh -c package'
+        sh 'sudo ./make_var_mx6ul_dart_debian.sh -c clean'
       }
     }
-    stage('Save Artifacts') {
-      steps {
-        archiveArtifacts(artifacts: 'output/*.deb', onlyIfSuccessful: true)
+
+    stage('Build Kernel and package') {
+      stages {
+        stage('Aventura') {
+          steps {
+            sh 'sudo ./make_var_mx6ul_dart_debian.sh -c package -t twonav-aventura-2018'
```

Comment:
If two different builds are made (trail && aventura), I think it might be safer to clean before each build. Am I correct?

Reply:
Although it doesnt create any issues now i am going to make the change to prevent future conflicts.